src/color-chart/color-chart.js

```diff
@@ -256,7 +276,15 @@ const Self = class ColorChart extends ColorElement {
 
 		yResolved: {
 			get () {
-				return Self.Color.Space.resolveCoord(this.y, "oklch");
+				try {
+					return Self.Color.Space.resolveCoord(this.y, "oklch");
+				}
+				catch {
```

Comment:
Why do we have this new error condition? I don't understand why this is here based on the comment below.

Reply:
Thank you for this question! You are absolutely right: We shouldn't handle invalid values of `y` here. I moved the logic to another place. Could you please take a look and see if it's more correct now?